lm-agent/lm_agent/server_interfaces/license_server_interface.py

```diff
@@ -1,13 +1,12 @@
 """Module for license server interface abstract base class."""
-import abc
 from typing import List
 
 from pydantic import BaseModel, Field
 
 from lm_agent.config import PRODUCT_FEATURE_RX
 
 
-class LicenseServerInterface(metaclass=abc.ABCMeta):
+class LicenseServerInterface:
```

Comment:
Why remove the ABC markers from this class?

Reply:
because it doesn't need to and mypy complains about it